### ISSUE TITLE
qt_lib.patch missing

### ISSUE THREAD

**Issue by sjewo** [JanMarvin/rstudio-archlinuxpkg#3]
You forgot to add  qt_lib.patch in your last commit bcd6f49a0d6a30e392754482129a549df7be72d9

**Comment by JanMarvin** [comment]
Thanks, for the report. Just pushed it.

**Comment by JanMarvin** [comment]
Was trying to fix https://github.com/rstudio/rstudio/issues/5961, but did not succeed :shrug: 
Still requires the `qt.conf` from [here](https://bugs.archlinux.org/task/64973?project=1&string=qt5-webengine)

Edit: Some google'ing showed that something like that has bitten Falkon users over at Gentoo iirc. Therefore I searched Falkons git history for things that changed and adopted the cmake build from Arch. That was not successful.
I assume that it is something internal to rstudio, when using or setting a Qt variable used to work for Qt < 5.14, but something was changed in Qt 5.14 and now it breaks. Using the `qt.conf` is a solution, but ultimately it might break something else.
